src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentMethodExampleFactory.php

```diff
@@ -26,8 +27,10 @@
  */
 class PaymentMethodExampleFactory extends AbstractExampleFactory implements ExampleFactoryInterface
 {
+    const DEFAULT_LOCALE = 'en_US';
```

Comment:
It always exist in database, if you have `Given the store operates on a single channel in "United States"` or `And it uses the "English (United States)" locale by default` or sth similar. It is not used in every feature which needs a translation.